_posts/0000-01-02-macks203.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
why did the chicken cross the road?, why the hell do you keep asking everyone but the chickens that question?